src/share/classes/com/sun/javatest/regtest/config/RegressionContext.java

```diff
@@ -101,6 +101,11 @@ static RegressionContext getDefault() {
         values.put("os.maxMemory", String.valueOf(os.maxMemory));
         values.put("os.maxSwap", String.valueOf(os.maxSwap));
 
+        String testThreadFactory = (params == null || params.getTestThreadFactory() == null)
+                ? "null"
```

Comment:
Having `null` as the value for "no test thread factory available" is right choice regarding easier handling on the use-site?

Or would skipping the `values.put()` call be a better default? 🤔

Reply:
I haven't checked but it should be considered as a "invalid name" if we don't put any value.
Also, want to be consistent with other properties, that just have some default value, often "null".